config/locales/dashboard.en.yml

```diff
@@ -322,8 +322,8 @@ en:
         complete: "Complete"
         actions: "Actions"
       form:
-        service: "Service:"
```

Comment:
Rather than adding asterisks to the YML, this should be done by adding the `required` class to the form labels